src/core.ts

```diff
@@ -135,7 +135,7 @@ function envelope(): IEnvelope {
   return {
     clarityId: cid,
     impressionId,
-    url: top.location.href,
+    url: location.href,
```

Comment:
Here is the more detailed support list. I don't think we should worry about IE < 9 at this point:
![image](https://cloud.githubusercontent.com/assets/23270423/26794708/a32260de-49d7-11e7-9b26-46d6e3193ec6.png)